sandbox/public/legacy_target.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="index.css" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src 'self' *.demdex.net;
+            script-src 'self' 'unsafe-inline' *.amazonaws.com github.com *.githubusercontent.com;
+            style-src 'self' 'unsafe-inline';
+            img-src * data:;
+            connect-src 'self' *.adobedc.net *.demdex.net *.tt.omtrdc.net"
+    />
+    <script src="/at.js"></script>
```

Comment:
I think we need Visitor.js as well so that all profile fragments are based on ECID right?

Reply:
Yes. The sandbox needs work to be able to test mixed mode. I'll do that in a separate PR.